homeassistant/components/nws/weather.py

```diff
@@ -253,7 +149,7 @@ def attribution(self):
     @property
     def name(self):
         """Return the name of the station."""
-        return self.station_name
+        return f"{self.station}_{self.mode}"
```

Comment:
```suggestion
        return f"{self.station} {self.mode.title()}"
```

The naming looks a bit off.  Suggest the above.

<img width="455" alt="Screen Shot 2020-04-07 at 6 56 36 PM" src="https://user-images.githubusercontent.com/663432/78730141-916d2980-7901-11ea-8dc3-318f6385439f.png">

Reply:
Once I wrote this out your suggestion, it reads and looks much better.